inc/3rd-party/plugins/advanced-custom-fields.php

```diff
@@ -0,0 +1,9 @@
+<?php
+defined( 'ABSPATH' ) || die( 'Cheatin\' uh?' );
+
+add_action( 'acf/save_post', 'rocket_clear_cache_on_acf_options_save' );
+function rocket_clear_cache_on_acf_options_save( $post_id ) {
```

Comment:
DocBlock?
That would explain for example why an argument named `$post_id` can be equal to the string `'options'` :D